srcpkgs/xorriso/template

```diff
@@ -1,12 +1,14 @@
 # Template file for 'xorriso'
 pkgname=xorriso
-version=1.5.3
+version=1.5.4.pl02
 revision=1
+_basever=1.5.4
```

Comment:
`_basever=${version%.pl*}`

Other than that, I think we're good with this update.

@ericonr  Anything I could miss?

Reply:
LGTM !